docker/bdd-tests/Dockerfile

```diff
@@ -0,0 +1,24 @@
+# Build for python API
+FROM python:3.8.5-buster as target
+
+ENV PYTHONDONTWRITEBYTECODE 1
+ENV PYTHONUNBUFFERED 1
+
+RUN apt-get update -y && \
+      apt-get -y install sudo
+
+RUN wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | sudo apt-key add -
+RUN sudo sh -c 'echo "deb [arch=amd64] http://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/google.list'
+RUN apt-get update -y
+RUN apt-get install -y google-chrome-stable
+
+
+WORKDIR /usr/src/app
+COPY . /usr/src/app
+
+RUN pip install --upgrade pip
+RUN pip install -r requirements.txt
+
+RUN sudo cp requirements/chromedriver /usr/bin
```

Comment:
is this sort of magic necessary? I.e. the installation script should be taking care of this sort of copy.

Reply:
Can we avoid distributing a binary, but pulling and installing it somehow?